static/js/common/upload-addon.js

```diff
@@ -317,12 +317,14 @@
 
             var $newForm = $('.new-addon-file');
             var $isUnlistedCheckbox = $('#id_is_unlisted');
```

Comment:
Since you removed that element from `StandaloneValidationForm` this can be removed as well.